src/xtgeo/well/wells.py

```diff
@@ -148,14 +148,15 @@ def from_files(
         # file checks are done within the Well() class
         for wfile in filelist:
             try:
-                wll = xtgeo.well.Well(
-                    wfile,
-                    fformat=fformat,
-                    mdlogname=mdlogname,
-                    zonelogname=zonelogname,
-                    strict=strict,
+                self._wells.append(
+                    xtgeo.well_from_file(
+                        wfile,
+                        fformat=fformat,
+                        mdlogname=mdlogname,
+                        zonelogname=zonelogname,
+                        strict=strict,
+                    )
                 )
-                self._wells.append(wll)
             except ValueError as err:
                 xtg.warn("SKIP this well: {}".format(err))
```

Comment:
Maybe outside the scope of this PR, but should we treat this differently? Seems a bit permissive to just allow the import to fail. If not fixing it here, create an issue.

Reply:
And see below that if we load no wells, we just get a warning.

Reply:
I'll create an issue :)

Reply:
https://github.com/equinor/xtgeo/issues/719